src/commands/api/validate.js

```diff
@@ -0,0 +1,46 @@
+const BaseCommand = require('../../support/command/base-command')
+const { getApiIdentifierArg } = require('../../support/command/parse-input')
+const { getApi } = require('../../requests/api')
+const { pipeAsync } = require('../../utils/general')
+const { getResponseContent } = require('../../support/command/handle-response')
+
+class ValidateCommand extends BaseCommand {
+  async run() {
+    const { args } = this.parse(ValidateCommand)
+    const apiPath = getApiIdentifierArg(args)
+    const validationResult = await this.getValidationResult(apiPath)
+    // eslint-disable-next-line immutable/no-let
+    let hasCritical = false
+    const validationResultsStr = validationResult.validation
+          .map(err => {
+            if (err.severity === 'CRITICAL') hasCritical = true
+            return `${err.line}: \t${err.severity} \t${err.description}`
+          })
+          .join('\n')
+    if (validationResult.validation.length) this.log('line \tseverity \tdescription\n')
+    this.log(validationResultsStr)
+    
+    if (hasCritical) this.exit(1)
+    this.exit(0)
+  }
+
+  getValidationResult(apiPath) {
+    return this.executeHttp({
+      execute: () => getApi([apiPath, 'validation']),
+      onResolve: pipeAsync(getResponseContent, JSON.parse),
+      options: {}
+    })
+  }
+}
+
+ValidateCommand.description = `get validation result of an API version
+The API version is required. An error will occur if the API version does not exist.
+`
+
+ValidateCommand.args = [{
+  name: 'OWNER/API_NAME/[VERSION]',
```

Comment:
You have the `VERSION` as an optional parameter - in other commands this means that if the VERSION is not included then you will apply the command to the default version of the API (see `api:get` as an example).

Reply:
Cool, I fix it as the version is required.

Reply:
I think allowing the version as optional is a nice feature. You would be first fetching the API's default version name and then using that in the validation request. You can copy the pattern used in the `api:get` command.